src/components/post-comment.jsx

```diff
@@ -72,6 +72,9 @@ export default class PostComment extends React.Component {
     if (shouldClearText) {
       this.setState({editText: ''});
     }
+    if (this.props.editText !== newProps.editText) {
```

Comment:
It's props all the time and it's like the only semantically correct option here, cause we don't really store actual commenting text in redux state.